python/src/deltachat/testplugin.py

```diff
@@ -178,6 +179,50 @@ def get_liveconfig_producer(self):
                     yield config
             pytest.fail("more than {} live accounts requested.".format(MAX_LIVE_CREATED_ACCOUNTS))
 
+    def cache_maybe_retrieve_configured_db_files(self, cache_addr, db_target_path):
```

Comment:
hum, i missed the point where we said everything should be done using type annotations.  I'd like to avoid adding them everywhere, to be honest.  The gain is orders of magnitude less than doing it with rust.